### DISCUSSION TITLE
Is influxdb2 supported?

### DISCUSSION THREAD

**Discussion by hailumeng** [karimra/gnmic#637] [Q&A] [answered]
I see example uses 1.8 format. But gnmic-config.yml refers bucket and token terms.

**Comment by karimra** [comment, marked as answer]
Yes 2.0 is supported, 
check this page for influxDB config options: https://gnmic.kmrd.dev/user_guide/outputs/influxdb_output/

**Reply by hailumeng** [reply]
Thank you Karimra! I will give it try and this makes things easier. 👍
